ci/unit/test_main_node.sh

```diff
@@ -51,4 +51,10 @@ else
 fi
 
 sudo mount -t tmpfs -o size=16G tmpfs /mnt/daos
-IS_CI=true RUN_TEST_VALGRIND="$WITH_VALGRIND" DAOS_BASE="$SL_SRC_DIR" utils/run_test.sh
+# Save the old command line to avoid merge conflicts when merging to other branches
+# that may have added more testing
+# IS_CI=true RUN_TEST_VALGRIND="$WITH_VALGRIND" DAOS_BASE="$SL_SRC_DIR" utils/run_test.sh
+RUN_TEST_VALGRIND=""
+if [ "$WITH_VALGRIND" = "memcheck" ]; then
```

Comment:
(lint) Couldn't find 'fi' for this 'if'. [SC1046]
(lint) Couldn't parse this if expression. Fix to allow more checks. [SC1073]